docs/usage/configuration/config_documentation.md

```diff
@@ -1584,7 +1584,8 @@ Config options related to Synapse's media store.
 ### `enable_media_repo`
 
 Enable the media store service in the Synapse master. Defaults to true.
-Set to false if you are using a separate media store worker.
+Set to false if you are using a separate media repository worker(and remember to set to
```

Comment:
This is the only place that got changed. Everything else is my editor happily removing whitespace without asking 🙄 .